wezterm-gui/src/termwindow/render/mod.rs

```diff
@@ -484,6 +484,8 @@ impl crate::TermWindow {
     }
 
     pub fn compute_cell_fg_bg(&self, params: ComputeCellFgBgParams) -> ComputeCellFgBgResult {
+        const CURSOR_MIN_CONTRAST: f32 = 2.5;
```

Comment:
I think we should make this an option because I just know that someone is going to complain about this otherwise, regardless of how well selected the default value might be!

I'd suggest adding an option named something like `cursor_minimum_contrast: f32` over here:

https://github.com/wezterm/wezterm/blob/b251509a2f9c488f5379a8457258eb0f07053fce/config/src/config.rs#L612-L613

You'll also need to add a little helper function to return the 2.5 default value and assign that using `#[dynamic(default = "default_contrast_ratio")]` as an attribute on that option.

then I think you should be good to examine `self.config.cursor_minimum_contrast` in this function instead of the constant.